src/shared/components/query/styles.module.scss

```diff
@@ -73,6 +73,11 @@
     color: $redError;
     font-weight: bold;
   }
+
+  .oqlMessage {
+    color: $blueNotice;

```

Comment:
i thought we were going to use the class text-info?

Reply:
I didn't want the other styling associated with it so I just copied the color

Reply:
ok, but in variables we should be able to reference the underlying bootstrap color name.  i think it's important to establish a practice wherein we standardize these sort of semantic color.